src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -138,20 +138,24 @@ protected void init(DerValue encoding, int req_type)
                         " req type is " + req_type);
 
                 System.out.println(">>> KDCRep: Message in bytes is =>");
-                byte[] dataBytes = encoding.getDataBytes();
-                StringBuilder sb = new StringBuilder();
-                for (int i = 0; i < dataBytes.length; i++) {
-                    if ((i % 16) == 0) {
-                        sb.append(String.format("%06X", i));
+                try {
```

Comment:
This seems inappropriate. Why not simply `if (dataBytes != null) { ... }` (which would also be consistent with later versions).